sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Helpers/ParameterLoaderTests.cs

```diff
@@ -76,6 +76,20 @@ public void SetParameterValues_WhenGivenSonarLintFileHasStringParameterType_Only
             analyzer.FlagsEnumNamePattern.Should().Be("1"); // value from XML file
         }
 
+        [TestMethod]
+        public void SetParameterValues_WhenGivenSonarLintFileHasBooleanParameterType_OnlyOneParameter_PopulatesProperty()
+        {
+            // Arrange
+            var options = CreateOptionsWithAdditionalFile("ResourceTests\\RuleWithBooleanParameter\\SonarLint.xml");
```

Comment:
I would rename `CreateOptionsWithAdditionalFile(string relativePath)` to `CreateOptions(string additionalFilePath)`. I like this more since it simplifies the method name and the parameter will provide all the relevant information about the context. I'm also thinking that if we add an additional parameter it's possible that the name will not make sense anymore and we will have to change it which make the code a bit harder to maintain.